scripts/build-local.py

```diff
@@ -110,6 +108,11 @@
         target_arch = build['target_arch']
         toolchain = build['toolchain']
 
+        if '-' in toolchain and not runtime:
```

Comment:
where does the `'-'` come from?

Reply:
I made a small comment about this above:

> so the error is made conditional on whether or not versioned toolchains (ones with a `-` in them) are actually requested.

Should I clarify with a code comment too?

Reply:
ah, in the PR description. Ok let me reparse what you said.

Reply:
yes, I think a comment in the code here would help clarify or at least help me remember what's going on here in the future.